Wabbajack.Common/Utils.cs

```diff
@@ -673,7 +666,9 @@ public static T Last<T>(this T[] a)
             return a[a.Length - 1];
         }
 
+        [return: MaybeNull]
```

Comment:
**Example of `MaybeNull` return attribute to power GetOrDefault Concepts**
Without the attribute, the code will complain when returning `default`.
However, if you swap `V` to be `V?`, then it requires you constrain it to either `class` or `struct`. 
The `MaybeNull` attribute is the way to get around this and set up a proper GetOrDefault setup